src/index.ts

```diff
@@ -1,8 +1,9 @@
 import { compile, registerHelper, RuntimeOptions } from 'handlebars';
 import { resolve } from 'path';
 import { Plugin as VitePlugin } from 'vite';
+import type { IndexHtmlTransformContext } from 'vite';
```

Comment:
I think this lines can be combined like so:

```suggestion
import { Plugin as VitePlugin, IndexHtmlTransformContext } from 'vite';
```

Reply:
I will change this, but using `import type` makes sure no unused imports are left over in the build js file.